recipes/assemblyutility/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "AssemblyUtility" %}
+{% set version = "20160209" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/yechengxi/AssemblyUtility.git
```

Comment:
You could use this tarball plus a checksum instead of git: https://github.com/yechengxi/AssemblyUtility/archive/8e476764eb67ebb486e673ecbd8a0fa5d8eae22c.tar.gz